esm/data.py

```diff
@@ -255,14 +257,17 @@ class BatchConverter(object):
     processed (labels + tensor) batch.
     """
 
-    def __init__(self, alphabet):
+    def __init__(self, alphabet, truncate: bool):
         self.alphabet = alphabet
+        self.truncate = truncate
 
     def __call__(self, raw_batch: Sequence[Tuple[str, str]]):
         # RoBERTa uses an eos token, while ESM-1 does not.
         batch_size = len(raw_batch)
         batch_labels, seq_str_list = zip(*raw_batch)
         seq_encoded_list = [self.alphabet.encode(seq_str) for seq_str in seq_str_list]
+        if self.truncate:
+            seq_encoded_list = [seq_str[:1022] for seq_str in seq_encoded_list]
```

Comment:
Don't hardcode this number, use `self. truncation_seq_length`

Reply:
I thought it was only useful to enforce sequences shorter than 1024. 
I changed the bool into an optional int as you suggested.